packages/fxa-auth-server/docs/swagger/oauth-server-api.ts

```diff
@@ -18,13 +18,13 @@ const OAUTH_SERVER_API_DESCRIPTION = {
       ## Errors
       Invalid requests will return 4XX responses. Internal failures will return 5XX. Both will include JSON responses describing the error.
 
-      **Example error:**
-      > {<br/>
-      >  \`"code": 400,\` // matches the HTTP status code<br/>
-      >  \`"errno": 101,\` // stable application-level error number<br/>
-      >  \`"error": "Bad Request",\` // string description of error type<br/>
-      >  \`"message": "Unknown client"\`<br/>
-      > }
+      **Example error:** \n
```

Comment:
Do we need the `\n` or can we add an extra empty line above the opening JSON[ish] blob?